flang/test/Lower/unstructured-loop.f90

```diff
@@ -0,0 +1,21 @@
+! RUN: bbc --always-execute-loop-body --emit-fir %s -o - | FileCheck %s
+
+! Given the flag `--always-execute-loop-body` the compiler emits an extra
+! code to change to tripcount, test tries to verify the extra emitted FIR.
+
+! CHECK-LABEL: func @_QPsome
+subroutine some()
+  integer :: i
+
+  ! CHECK: [[tripcount:%4]] = divi_signed %3, %c1_i32_0 : i32
```

Comment:
The hardcoded labels make this test future-brittle. The slightest change means this test will fail.

Reply:
A CHECK line of

`%[[tripcount:.*]] = divi_signed`

should be sufficient here. As written this must be ssa-value `%4`, etc. Since it is possible for instruction orders to change, this may fail to match.